### ISSUE TITLE
Add `double` support to the hex abstraction/conversion overlay

### ISSUE THREAD

**Issue by MrGcGamer** [NationalSecurityAgency/ghidra#3689]
**Is your feature request related to a problem? Please describe.**
I mainly reverse Obj-C code, which often uses doubles for various stuff like color values etc.
Ghidra will give me a pseudo code output along the lines of:
```
  _objc_msgSend(0x3ff0000000000000,0x3ff0000000000000,0,0x3ff0000000000000,&_OBJC_CLASS_$_UIColor,
                s_colorWithDisplayP3Red:green:blue_1d9795b38);
```
and `0x3ff0000000000000` ain't that useful to me, especially, when the numbers become more precise.

**Describe the solution you'd like**
Therefor it would be really great, if the popup menu on right-click would also allow the conversion to `double` alongside:
 - Binary
 - Decimal
 - Octal
 - Char
,which are currently available.

**Additional context**
Basically another option in that list of conversion options
![Screenshot 2021-11-30 at 19 58 18](https://user-images.githubusercontent.com/32813275/144110459-97ca5a19-7a67-46c4-b480-e1b55dacbcfe.png)
